__tests__/app.test.js

```diff
@@ -67,6 +67,130 @@ describe("/api/reviews", () => {
                 });
             });
     });
+    test("GET - 200: Responds with an array of review objects sorted by date in descending order if no query params are added", () => {
+        return request(app)
+            .get("/api/reviews")
+            .expect(200)
+            .then((res) => {
+                const reviews = res.body.reviews;
+                expect(reviews.length).toBeGreaterThan(0);
+                reviews.forEach((review) => {
+                    expect(review).toMatchObject({
+                        review_id: expect.any(Number),
+                        title: expect.any(String),
+                        category: expect.any(String),
+                        designer: expect.any(String),
+                        owner: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number),
+                    });
+                });
+                expect(reviews).toBeSortedBy("created_at", {
+                    descending: true,
+                });
+            });
+    });
+    test("GET - 200: Responds with an array of review objects containing only the 'social deduction' category sorted by date in descending order if the category query param was requested", () => {
+        return request(app)
+            .get("/api/reviews?category=social+deduction")
```

Comment:
You dont need to manually add the + here, you can have it with a space here and that would be handled for you.